client/blocks/jitm/templates/sidebar-banner.jsx

```diff
@@ -32,7 +31,7 @@ export default function SidebarBannerTemplate( {
 				href={ CTA.link }
 				onClick={ onClick }
 				onDismissClick={ onDismiss }
-				title={ preventWidows( message ) }
```

Comment:
Not sure how to test this one, but it uses `UpsellNudge` which uses `Banner` which wraps its title in a `p` tag so is covered by generic `p` styles.

Reply:
I believe it's this sidebar upsell that you see on a Free plan from its Home screen. I notice with this PR applied that the `banner__title`'s `text-wrap: wrap` rule takes precedence over the generic `p` rule, so we wind up with a widow:

<img width="1182" alt="image" src="https://github.com/user-attachments/assets/1c9c2169-7072-45b7-b161-8c227cf48112" />

Reply:
Oooh good catch! That rule is so specific that I think we can safely remove it.

Reply:
Ok done!